### PR TITLE
Bad actor with IP violation and rug pull

### PR DESCRIPTION
**Pull request by kaimeinke**
This actor 0x2D3fb3096F020029A49A45c88253A4CD0BC2503F pulled data from Kaggle (URI: https://www.kaggle.com/gsutters/the-human-freedom-index?select=hfi_cc_2018.csv) without giving the proper license information on did:op:3CeA3612EA153a6b8Ce269E6a93E797EE73214F1 and performed a rug pull.

### CODE REVIEW

**Comment by kaimeinke** [comment]
Meanwhile the owner of the pool has confirmed the rugpull was intentional and the community has discussed the issue with the content provider after investigation by Agent_K.

This has revealed that other addresses were used to create others sets that were rugpulled by the same actor. 

Other addresses will need to be added with another pull request later.

Screenshot_20201129-155944_Telegram

**Comment by kaimeinke** [comment]
The new address 0x359a76dB7aFDAd46f0e2C6699eC768c1064C86Ca has been added with a new commit. This is the creator adress for did:op:6Ee6f092229E32c708569f98Ad0E6A7206371E94

This is the transaction that links both accounts. https://etherscan.io/tx/0x637ccbb860a1ebfe0131d224280ca0c9972a46d3241490748c78d16399e2bb77

**Comment by kaimeinke** [comment]
Will this request go any further or did I do something wrong that blocks a merge? 

Asset https://market.oceanprotocol.com/asset/did:op:3CeA3612EA153a6b8Ce269E6a93E797EE73214F1 is still live.
Asset https://market.oceanprotocol.com/asset/did:op:6Ee6f092229E32c708569f98Ad0E6A7206371E94 is still live.

Both belong in purgatory.

File: list-accounts.json [hunk 1]
```diff
@@ -683,4 +683,12 @@
         "address": "0xd51096feB90de6Dc60e318Bf27fD9432051EBfcd",
         "reason": "violation"
     }
```
Comment (kremalicious) [inline list-accounts.json]: needs a comma here

```suggestion
    },
```

File: list-accounts.json [hunk 2]
```diff
@@ -683,4 +683,12 @@
         "address": "0xd51096feB90de6Dc60e318Bf27fD9432051EBfcd",
         "reason": "violation"
     }
+    {
+        "address": "0x2D3fb3096F020029A49A45c88253A4CD0BC2503F",
+        "reason": "bad actor and ip violation"
+    }
```
Comment (kremalicious) [inline list-accounts.json]: and here too

```suggestion
    },
```

**Comment by kremalicious** [comment]
process note: putting an account into purgatory will not automatically put its assets into purgatory. If this is desired, `list-assets.json` would need to be edited as part of this PR too.

**Comment by kaimeinke** [comment]
Thanks a lot for the help @kremalicious , I should have seen this and now I will look out for it. I will add the assets to `list-assets.json` as well.

**Review by daneklustig — CHANGES_REQUESTED** [review]
Please use one of the categories for putting datasets into purgatory: rug pull / sensitive data / impersonation / ip infringement / violation

**Review by daneklustig — APPROVED** [review]

**Pull request merged**